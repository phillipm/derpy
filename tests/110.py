a = 1 + 1
b = 2 % 3
c = 1 ^ (x-3)*1 & 2 ^ 5
d = 1 - 1
f = 1 * 1
g = 1 / 1
