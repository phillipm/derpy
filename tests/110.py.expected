(program
 (= (a) (arith 1 ("+" 1)))
 (= (b) (term 2 ("%" 3)))
 (= (c) (bitwise-xor 1 (bitwise-and (term (arith x ("-" 3)) ("*" 1)) 2) 5))
 (= (d) (arith 1 ("-" 1)))
 (= (f) (term 1 ("*" 1)))
 (= (g) (term 1 ("/" 1))))

